docs: adding new options to write/read parts of docs (#1076)
neo4j-contrib/neo4j-spark-connector: docs/modules/ROOT/pages/read/options.adoc

|_(empty)_
|Yes

4+|*Cypher options* [role=label--new-6.0]

|`cypher.version`
|Set a custom Cypher language version to a custom value.

> Expected value is the version number itself, e.g. `5` or `25`

Reviewer: ```suggestion
The expected value is the version number itself, for example `5` or `25`.
```

|5
|No

|`cypher.tuning.*`
|Set any cypher query tuning parameter with this prefix.

> See details on how to properly use this feature link:{neo4j-docs-base-uri}/spark/current/performance/cypher-tuning[here]

Reviewer: ```suggestion
See more details on how to properly use this feature on link:{neo4j-docs-base-uri}/spark/current/performance/cypher-tuning[].
```

|_(emtpy)_
|No

4+|*Optimization options*

|`script`
|Apply a custom Cypher query which will run once before any write operations.
Use this capability to apply custom optimizations e.g. custom index/contstraint queries.

> Can not be used with `script.N`, use either or.

Reviewer: ```suggestion
|Apply a custom Cypher query which will run once before any write operations.
Use this option to apply optimizations such as the use of a  custom index and constraint queries.
Can not be used together with the numbered `script.N` option.
```

|_(empty)_
|No

|`script.N`
|Similar to `script`, replace `N` with any positive integer, put as many as you like.
The scripts are executed in succession and in numbered order.

> Can not be used with plain `script`, use either or.

Reviewer: ```suggestion
Can not be used together with the unnumbered `script` option.
```
